script.js

```diff
@@ -0,0 +1,80 @@
+window.onload = startGame;
+
+function startGame() {
+  //when clearBoxes is called, clears all 9 boxes.
+  for (var i=1; i<=9; i = i+1) {
+    clearBoxes(i);
+  }
+  //by default game starts with "X" moving. will check to see if i can reset that each turn.
+  document.turn = "X";
```

Comment:
why are we attaching property's to the global `document` object? How could we create our own `game` object?